src/components/ProjectSearch/textSearch/utils/highlight.js

```diff
@@ -0,0 +1,21 @@
+// Maybe reuse file search's functions?
+import React from "react";
+
```

Comment:
You would need to add 
```js 
highlightMatches.displayName = "HighlightMatches";
```
 here to fix the lint issue,

 because react sees this as a stateless component.